packages/client/hmi-client/src/components/workflow/tera-chart-control.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+	<div class="chart-control">
+		<div class="multiselect-container">
+			<MultiSelect
+				v-model="selectedVariable"
+				:options="variables"
+				placeholder="What do you want to see?"
+				@update:model-value="updateSelectedVariable"
+				filter
+			>
+				<template v-slot:value>
+					<template v-for="(variable, index) in selectedVariable" :key="index">
+						<template v-if="index > 0">,&nbsp;</template>
+						<span> {{ variable }} </span>
+					</template>
+				</template>
+			</MultiSelect>
+			<Button
+				v-if="showRemoveButton"
+				title="Remove chart"
+				icon="pi pi-trash"
+				@click="$emit('remove')"
+				rounded
+				text
+			/>
+		</div>
+	</div>
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue';
+import Button from 'primevue/button';
+import MultiSelect from 'primevue/multiselect';
+import { ChartConfig } from '@/types/SimulateConfig';
+
+const props = defineProps<{
+	variables: string[];
+	chartConfig: ChartConfig;
+	showRemoveButton: boolean;
+}>();
+
+const emit = defineEmits(['configuration-change', 'remove']);
+
+const selectedVariable = ref<string[]>(props.chartConfig.selectedVariable);
+
+const updateSelectedVariable = () => {
+	emit('configuration-change', {
+		selectedVariable: selectedVariable.value,
+		selectedRun: props.chartConfig.selectedRun
+	});
+};
+</script>
+
+<style scoped>
+.chart-control {
+	background-color: var(--surface-0);
+	border: 1px solid var(--surface-border-light);
+	border-radius: var(--border-radius);
+	padding: var(--gap-small);
+	position: relative;
+}
+
+.multiselect-title {
+	font-size: smaller;
+	font-weight: 700;
+}
+.multiselect-container {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	gap: 0.5rem;
```

Comment:
https://github.com/DARPA-ASKEM/terarium/blob/d53387d84a4568a4565a6cb6a22c0a569ffdf2e3/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L984C2-L984C9
```suggestion
	gap:var(--gap-2);
```